### PR TITLE
Purge relative times

### PR DESCRIPTION
**Pull request by JoshKarpel**
This PR replaces language like "today", "tomorrow", or "yesterday" with "previous" or "next", or similar. This is better when we don't necessarily know when learners will actually reach any given exercise.

### CODE REVIEW

File: docs/materials/osg/ex1-submit-refresher.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ Now, let’s try to remember some basic HTCondor ideas from earlier today:
 
         Requirements = (HAS_CVMFS_oasis_opensciencegrid_org =?= TRUE)
 
-    Try to do this step without looking at materials from earlier today.
-    But if you are stuck, see [today’s exercise 2.2](/materials/htc/part2-ex2-queue-n.md).
+    Try to do this step without looking at materials from the earlier exercises.
+    But if you are stuck, see [Exercise 2.2](/materials/htc/part2-ex2-queue-n.md).
```
Comment (osg-cat) [inline docs/materials/osg/ex1-submit-refresher.md]: “Exercise 2.2” is technically (and actually) not a unique identifier — probably needs to be “HTC Exercise 2.2”.

File: docs/materials/osg/ex2-login-scp.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ critical for all remaining exercises.
 Log in to the OSG submit server
 --------------------------------
 
-For some of the remaining exercises today, you will be using a server named `login04.osgconnect.net`.
+For some of the remaining exercises in this section, you will be using a server named `login04.osgconnect.net`.
```
Comment (osg-cat) [inline docs/materials/osg/ex2-login-scp.md]: “in this topic”

File: docs/materials/osg/ex5-software-diffs.md [hunk 3]
```diff
@@ -12,10 +12,10 @@ software you need won't even be installed.
 Refresher - condor\_status
 --------------------------
 
-The OSG pool, like the local pool you used earlier today, is just another HTCondor pool.
+The OSG pool, like the local pool you used earlier, is just another HTCondor pool.
 This means that the commands you use will be the same and the jobs you submit can have similar payloads but there is one
 major difference: the slots are different!
-You can use the `condor_status` command just as you did yesterday to inspect these differences.
+You can use the `condor_status` command to inspect these differences.
```
Comment (osg-cat) [inline docs/materials/osg/ex5-software-diffs.md]: As before, you can use…

**Comment by JoshKarpel** [comment]
Review comments resolved

**Pull request merged**